pre_build/pre-build-job.yml

```diff
@@ -0,0 +1,76 @@
+- default:
+    name: global
+    prebuild_render: "git@github.com:bamachrn/cccp-pre-build-code.git"
```

Comment:
Are we sure about the ssh key configuration? Asking since we're using `git` based url instead of `https`.

Reply:
yes, we need to push the changes to the repository. its best to use `ssh key` for the same which is locally configured in the jenkins `ci-slave`. as we can not use access tokens due to security issues.